package.json

```diff
@@ -33,7 +34,7 @@
 		"storybook:build": "build-storybook -c .storybook -o .storybook-static",
 		"storybook": "start-storybook -p 8888",
 		"test": "jest",
-		"version": "yarn run format:all"
+		"version": "yarn genGlobalChangeLog && prettier --write -- \"packages/**/CHANGELOG.md\" && git add -- \"CHANGELOG.md\" \"packages/clay-css/src/scss/_license-text.scss\""
```

Comment:
I had to add these `git add` commands so that lerna would include the changes to the CHANGELOG to the publish commit. I was hoping to use [their config option](https://github.com/lerna/lerna/blob/main/commands/version/README.md#--no-granular-pathspec) for this, but I wasn't able to get it working properly.

The license file was an issue we needed to address and it was the same solution as the CHANGELOG so I included that here.

Reply:
Oh well, at least it is a very specifically targeted `git add`, so it's unlikely to do anything nefarious.